intelmq/bots/parsers/shadowserver/_config.py

```diff
@@ -204,6 +206,22 @@ def convert_date_utc(value: str) -> Optional[str]:
     return harmonization.DateTime.sanitize(value + '+00:00')
 
 
+def force_base64(value: Optional[str]) -> Optional[str]:
+    """
+    Takes input strings that may be either base64-encoded bytestrings or plaintext string,
+    and leaves the base64-encoded values untouched while encoding the non-encoded values,
+    uniformly converting the data in the field to be base64-encoded
+    """
+    if not value:
+        return None
+
+    try:
+        base64.b64decode(value)  # return value intentionally ignored
+        return value
+    except binascii.Error:
+        return base64.b64encode(value.encode()).decode()
```

Comment:
let's keep the try-block minimal to avoid unexpected behaviour (although the return statement shouldn't cause any direct troubles, having return inside try is not a good practice)

```suggestion
    try:
        base64.b64decode(value)  # return value intentionally ignored
    except binascii.Error:
        return base64.b64encode(value.encode()).decode()
    else:
        return value
```